exercises/practice/diamond/diamond_test.cpp

```diff
@@ -0,0 +1,53 @@
+	REQUIRE(diamond::rows('D') == expected);
+}
+
+TEST_CASE("Largest possible diamond", "[82ea9aa9-4c0e-442a-b07e-40204e925944]") {
+	std::vector<std::string> expected{"                         A                         ", "                        B B                        ", "                       C   C                       ", "                      D     D                      ", "                     E       E                     ", "                    F         F                    ", "                   G           G                   ", "                  H             H                  ", "                 I               I                 ", "                J                 J                ", "               K                   K               ", "              L                     L              ", "             M                       M             ", "            N                         N            ", "           O                           O           ", "          P                             P          ", "         Q                               Q         ", "        R                                 R        ", "       S                                   S       ", "      T                                     T      ", "     U                                       U     ", "    V                                         V    ", "   W                                           W   ", "  X                                             X  ", " Y                                               Y ", "Z                                                 Z", " Y                                               Y ", "  X                                             X  ", "   W                                           W   ", "    V                                         V    ", "     U                                       U     ", "      T                                     T      ", "       S                                   S       ", "        R                                 R        ", "         Q                               Q         ", "          P                             P          ", "           O                           O           ", "            N                         N            ", "             M                       M             ", "              L                     L              ", "               K                   K               ", "                J                 J                ", "                 I               I                 ", "                  H             H                  ", "                   G           G                   ", "                    F         F                    ", "                     E       E                     ", "                      D     D                      ", "                       C   C                       ", "                        B B                        ", "                         A                         "};
```

Comment:
Would it be possible to format the expected result in a way that the "diamond" can be seen in the source code?

Reply:
Great idea. That would be nice, I will try that and see if it reaches the user in the web editor.

Reply:
Just to avoid a misunderstanding: I meant reformatting the source code, each element (line) of `expected` on its own line so that the pattern is visible in the code.
Changing how a failed test gets displayed would be nice but I suspect that would be much harder.